code/modules/mining/ores_coins.dm

```diff
@@ -136,10 +136,10 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 
 /obj/item/stack/ore/glass/basalt
 	name = "volcanic ash"
+	desc = "Looks like you could shove some on a girder to make a false rock wall."
```

Comment:
this is a bit in your face, maybe move this desc to examine_more?

Reply:
I'll cook something up. I just copied that one from the asteroid rock dust, which shares the same description.